style.css

```diff
@@ -478,7 +475,178 @@ textarea:focus {
   font-size: 32px;
 }
 
```

Comment:
- [ ] Please make sure to apply all the requirements asked by the previous reviewer in this [comment](https://github.com/Yoz21/Portfolio/pull/12#discussion_r792700704
). Here the reviewer asked for implementing a blurred background once the popup is done, so please make sure to apply it! You can use the `filter` CSS property for that.
- [ ] [OPTIONAL] Also, it would be great to make the popup look good on large screens. Right now, when you open the portfolio on a large screen, the popup does not show completely, so it can lead to a bad user experience.